src/dataCommand.ts

```diff
@@ -63,26 +70,26 @@ export abstract class DataCommand extends SfdxCommand {
 
   public validateIdXorWhereFlags(): void {
     if (!this.flags.where && !this.flags.sobjectid) {
-      throw new SfdxError(messages.getMessage('NeitherSobjectidNorWhereError'), 'NeitherSobjectidNorWhereError', [
+      throw new SfError(messages.getMessage('NeitherSobjectidNorWhereError'), 'NeitherSobjectidNorWhereError', [
         messages.getMessage('NeitherSobjectidNorWhereErrorActions'),
       ]);
     }
   }
 
-  public collectErrorMessages(result: ErrorResult): string {
+  public collectErrorMessages(result: SaveResult): string {
     let errors = '';
     if (result.errors) {
       errors = '\nErrors:\n';
-      result.errors.forEach((err) => {
-        errors += '  ' + err + '\n';
+      result.errors.map((err) => {
+        errors += '  ' + err?.message + '\n';
       });
     }
     return errors;
   }
 
   public async throwIfFileDoesntExist(path: string): Promise<void> {
-    if (!(await fs.fileExists(path))) {
-      throw new SfdxError(messages.getMessage('PathDoesNotExist', [path]), 'PathDoesNotExist');
+    if (!(await fse.pathExists(path))) {
```

Comment:
Curious about the distinction in method names... calling method refers to `throwIf**File**DoesntExist`, but the param and implementation both refer to `path`. Those are pretty different things. Could this lead to confusion by the caller?